From 670eca9665b3e865f9d7ffe2b97cbde036dd90a6 Mon Sep 17 00:00:00 2001
From: Ken Ma <make@marvell.com>
Date: Wed, 20 Sep 2017 16:04:29 +0800
Subject: [PATCH 1240/1345] doc: mvebu: update a37x0 drivers' binding files in
 porting guide

commit  86f0e66d46a931c8e52b1450ddd9ee47eb0ce6dd from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

Update a37x0 drivers' binding files in porting guide:
- GPIO driver is integrated into pinctrl driver, they share the binding
  file "pinctrl/marvell,armada-37xx-pinctrl.txt";
- Clock driver is split to 3 drivers: periph clock driver, tbg
  clock driver and xtal clock driver, the binding files are updated
  correspondingly;
- Marvell A37x0 USB PHY driver now uses the binding file
  "usb/armada3700-usb-phy.txt".

Change-Id: Idf5857684178836b1cf0c1023ebff8e64f2ca498
Signed-off-by: Ken Ma <make@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/44663
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hua Jing <jinghua@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 Documentation/mvebu/porting_guide.txt |   10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/Documentation/mvebu/porting_guide.txt b/Documentation/mvebu/porting_guide.txt
index 2b2b23f..7dde9cb 100644
--- a/Documentation/mvebu/porting_guide.txt
+++ b/Documentation/mvebu/porting_guide.txt
@@ -125,7 +125,7 @@ A37x0
 	+---------------------------------------+-----------------------------------------------+
 	| Driver				| Binding file					|
 	+=======================================+===============================================+
-	| Marvell A37x0 Pinctrl driver		| pinctrl/marvell,armada-3700-pinctrl.txt	|
+	| Marvell A37x0 Pinctrl & GPIO driver	| pinctrl/marvell,armada-37xx-pinctrl.txt	|
 	+---------------------------------------+-----------------------------------------------+
 	| Marvell Neta Ethernet controller	| net/marvell-armada-370-neta.txt		|
 	+---------------------------------------+-----------------------------------------------+
@@ -135,13 +135,13 @@ A37x0
 	+---------------------------------------+-----------------------------------------------+
 	| Marvell A37x0 UART driver		| tty/serial/mvebu-uart.txt			|
 	+---------------------------------------+-----------------------------------------------+
-	| Marvell A37x0 Clock driver		| clock/mvebu-armada3700-clock.txt		|
-	+---------------------------------------+-----------------------------------------------+
-	| Marvell A37x0 GPIO driver		| gpio/gpio-armada-3700.txt			|
+	| Marvell A37x0 Clock driver		| clock/armada3700-periph-clock.txt		|
+	|                           		| clock/armada3700-tbg-clock.txt		|
+	|                           		| clock/armada3700-xtal-clock.txt		|
 	+---------------------------------------+-----------------------------------------------+
 	| Marvell A37x0 PCIe driver		| pci/aardvark-pci.txt				|
 	+---------------------------------------+-----------------------------------------------+
-	| Marvell A37x0 USB OTG PHY driver	| usb/armada3700-otg-phy.txt			|
+	| Marvell A37x0 USB PHY driver		| usb/armada3700-usb-phy.txt			|
 	+---------------------------------------+-----------------------------------------------+
 
 A39x
-- 
1.7.9.5

